nvidia-device-plugin.yml

```diff
@@ -8,22 +8,17 @@ spec:
     type: RollingUpdate
   template:
     metadata:
-      # Mark this pod as a critical add-on; when enabled, the critical add-on scheduler
-      # reserves resources for critical add-on pods so that they can be rescheduled after
-      # a failure.  This annotation works in tandem with the toleration below.
-      annotations:
```

Comment:
Do you mind linking to the change / docs that mention this depreciation?

What happens when we have this Annotation in the newer versions?

Reply:
Done.

The annotation will just be ignored and the pod's priority will be 0 (meaning low priority)

Reply:
Can you keep this annotation

Reply:
Done